tensorflow/contrib/hvx/hexagon_controller/src_impl/hexagon_controller.c

```diff
@@ -19,7 +19,7 @@ limitations under the License.
 
 #include "hexagon_controller.h"
 
-#include <malloc.h>
```

Comment:
The [malloc.h](https://github.com/google/gemmlowp/blob/6a2a90822e8546fc2bfa7044de0faf1c1cb4862f/internal/platform.h#L28) should be declared by stdlib.h (See [malloc](https://linux.die.net/man/3/malloc) man page].